workspaces/src/network/variants.rs

```diff
@@ -72,7 +94,7 @@ where
         Ok(res)
     }
 
-    pub async fn dev_generate(&self) -> (AccountId, SecretKey) {
+    pub async fn generate_tla_credentials(&self) -> (AccountId, SecretKey) {
```

Comment:
should we rename it to generate_sponsored_credentials?

Reply:
It will probably get confusing inasmuch as we have the following two unrelated to this function  `create_sponsored_account` and `create_sponsored_account_and_deploy`

Reply:
fair